typescript/web/src/components/website/Hero/Hero.tsx

```diff
@@ -128,27 +120,21 @@ export function Hero() {
+          <Flex
+            direction="column"
+            justify="center"
+            flexGrow={1}
             pos="relative"
-            w={{ base: "full", lg: "560px" }}
+            MaxH="0"
+            minW={{ base: "full", lg: "560px" }}
             h={{ base: "auto", lg: "560px" }}
           >
-            <Img
-              w="full"
-              transform={{ base: "", lg: "scale(1.1)", xl: "scale(1.3)" }}
-              pos="relative"
-              zIndex="1"
-              rounded={8}
-              h={{ lg: "100%" }}
-              objectFit="contain"
-              backgroundSize="contain"
-              backgroundRepeat="no-repeat"
-              backgroundPosition="center"
-              backgroundImage="url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAMCAYAAABiDJ37AAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAAhGVYSWZNTQAqAAAACAAFARIAAwAAAAEAAQAAARoABQAAAAEAAABKARsABQAAAAEAAABSASgAAwAAAAEAAgAAh2kABAAAAAEAAABaAAAAAAAAAEgAAAABAAAASAAAAAEAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAFKADAAQAAAABAAAADAAAAAARd0G9AAAACXBIWXMAAAsTAAALEwEAmpwYAAABWWlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNi4wLjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgIDwvcmRmOkRlc2NyaXB0aW9uPgogICA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgoZXuEHAAAC/0lEQVQoFT2T228bRRSHv5mdvTi72cbBdkSahKaXtJGCQCBEpbYSqnjhGSHxhMQ7/0zFE38DUl/7UAkkriKoagUvIY2TJhG1Q5rN2mvHa3tmh0momNHcjmbmnPmdbwSvSzfLPvel93Wg1JFSvi+w/1ULlWtSiIudla3+n8O5zeJJz0hPzmpttlS3KFoLSTHI+97aXJo2ynHZCEKF1prtvQ7jqeZsWFDpMYmaMOifEPgh0vOJ4jn2dw8xImB3+wU3128IcdrvP7Na3DTWn/bPTFyMRGWFENaFZaspwh20laEyxkXmInWdMdo5tKSJZLHhsfP8L/1k80l449bGtsjy/LA3vbT0+E/Ni1P4rWNZiiY0R22uTHfYmr3PoU5J5dRFJVES6jXItODusuWzD2Em8o0ze5OKtlKeN+pk8GvPTt+eHfqfhl2SMEP2dohetUkWmzTMu2hCYjkhDKRrHquiYmm+choqfvjue/vs8UPufPwJygqlli5N+GJ5V6bBK0bViCpKKbw6utfgWmvMitim721gvNglACLOmFEhrdhdKCRx6PPOSkooLSIfjtv50f7VzV8emdn6vGeF00wFlN1D5Ms9vOvvMzBjRu7ZxaSFLAtW0yNe9mOaK2t8dG+dpBZoX3mqnJi28pzI+UDz7WbB1ZbEjgdUYc15OuaWOeZ5v03g9LusOtSMYH+0RhkXPD0O+SBZuEBoUE5treZRTqaVstZiHEuH+CyHMaX1GTvuLgeKhnvWXDykV6Vk8jZZuMDBoKJrCs4WU+J09oJV151jizZaKuEFxAF82fwdTIPT04zVuu+kOYZhjz/yu3zjMp9HA0p9wpsMWGs22TvLuP9WdKGh48pl4hx+KZUZ5k7ckOtX1pFJg+i9OU46f5OPFbVajTfMAl+VFVYbB3aFcaMjg85UMT+f6Lk4VP1i9OCgc/qgHvmp6m49ZVCO6M9scOf2PeJ6i59+/pG1ZIbQZU/mgth3qDjIbRCinMahGJOphMitj/8p8JQ8uLY4f3D+N/8FfMZadBxcaqEAAAAASUVORK5CYII=')"
-              src="/static/img/home-screenshot5.png"
-              alt="LabelFlow"
-            />
-          </Box>
+            <AspectRatio ratio={16 / 9}>
+              <video playsInline autoPlay muted loop>
+                <source src="/static/img/home-video.webm" type="video/webm" />
+              </video>
+            </AspectRatio>
```

Comment:
Can we move it into a `HomeVideo` component declared on top of this file?

Reply:
Done